WPMedia/fusion-news-theme-blocks
# THEMES-1173: header nav | Update button component to match new design

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description

  -  Fixes logo hover aplha to 80%
  -  Includes changes for links when hover

## Jira Ticket

- [THEMES-1173](https://arcpublishing.atlassian.net/browse/THEMES-1173)

## Acceptance Criteria

- hover state: opacity change on the logo to 70%
- button component matches design reqs (button group 1 as default)
- the behavior of links in the nav bar should be the same as the buttons. 

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout hotfix/THEMES-1173`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/header-nav-chain-block`
3. Go to a page with the nav block for example http://localhost/pf/all-blocks-test-ta/?_website=daily-intelligencer
4. Interact with the links in the nav block
5. See state hover

## Effect Of Changes

### Before

- When hovering the logo alpha is 70%.
- When hovering links color changes to black

https://github.com/WPMedia/arc-themes-blocks/assets/133056774/ae3f22fb-36cc-4d33-b1da-f62e6ef1bf29


### After

- When hovering the logo alpha is 80%.
- When hovering links color changes to global-neutral-2

https://github.com/WPMedia/arc-themes-blocks/assets/133056774/2fb48725-8872-4814-bd34-9780f989d839


## Dependencies or Side Effects

_Examples of dependencies or side effects are:_

- Feature pack pr for local development:
- How to deploy to news: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3138682964/News+Theme+2.0+Migration+Development+Deployment+Notes
k
## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1173]: https://arcpublishing.atlassian.net/browse/THEMES-1173?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
